pennylane_lightning/lightning_qubit.py

```diff
@@ -34,33 +33,22 @@
 from ._version import __version__
 
 try:
-    if platform.system() == "Windows" and sys.version_info[:2] >= (3, 8):  # pragma: no cover
-        # Add the current directory to DLL path.
-        # See https://docs.python.org/3/whatsnew/3.8.html#bpo-36085-whatsnew
-        os.add_dll_directory(os.path.dirname(os.path.abspath(__file__)))
-        from lightning_qubit_ops import (
-            StateVectorC64,
-            AdjointJacobianC64,
-            VectorJacobianProductC64,
-            StateVectorC128,
-            AdjointJacobianC128,
-            VectorJacobianProductC128,
-        )
-    else:
-        from .lightning_qubit_ops import (
-            StateVectorC64,
-            AdjointJacobianC64,
-            VectorJacobianProductC64,
-            StateVectorC128,
-            AdjointJacobianC128,
-            VectorJacobianProductC128,
-        )
+    from .lightning_qubit_ops import (
+        StateVectorC64,
+        AdjointJacobianC64,
+        VectorJacobianProductC64,
+        StateVectorC128,
+        AdjointJacobianC128,
+        VectorJacobianProductC128,
+    )
+
     from ._serialize import _serialize_obs, _serialize_ops
 
     CPP_BINARY_AVAILABLE = True
-except ModuleNotFoundError:
+except (ModuleNotFoundError, ImportError):
     CPP_BINARY_AVAILABLE = False
```

Comment:
I am guessing the tiny drop in coverage is from here. Maybe adding a quick test to check for the expected exception types may fix the drop, and all the other tests to continue/pass. What do you think?

Reply:
Hi Lee, as this became a part of #202, I will close the PR. I was using this PR as a test purpose :upside_down_face: